parcels/tools/statuscodes.py

```diff
@@ -29,62 +32,50 @@ class StatusCode:
 class DaskChunkingError(RuntimeError):
     """Error indicating to the user that something with setting up Dask and chunked fieldsets went wrong."""
 
-    def __init__(self, src_class_type, message):
-        msg = f"[{src_class_type}]: {message}"
-        super().__init__(msg)
+    pass
 
 
 class FieldSamplingError(RuntimeError):
     """Utility error class to propagate erroneous field sampling."""
 
-    def __init__(self, z, y, x, field=None):
-        self.field = field
-        self.x = x
-        self.y = y
-        self.z = z
-        message = f"{field.name if field else 'Field'} sampled at (depth={self.z}, lat={self.y}, lon={self.x})"
-        super().__init__(message)
+    pass
 
 
 class FieldOutOfBoundError(RuntimeError):
     """Utility error class to propagate out-of-bound field sampling."""
 
-    def __init__(self, z, y, x, field=None):
-        self.field = field
-        self.x = x
-        self.y = y
-        self.z = z
-        message = (
-            f"{field.name if field else 'Field'} sampled out-of-bound, at (depth={self.z}, lat={self.y}, lon={self.x})"
-        )
-        super().__init__(message)
+    pass
 
 
 class FieldOutOfBoundSurfaceError(RuntimeError):
     """Utility error class to propagate out-of-bound field sampling at the surface."""
 
-    def __init__(self, z, y, x, field=None):
-        self.field = field
-        self.x = x
-        self.y = y
-        self.z = z
-        message = f"{field.name if field else 'Field'} sampled out-of-bound at the surface, at (depth={self.z}, lat={self.y}, lon={self.x})"
-        super().__init__(message)
+    pass
+
+
+def _raise_field_sampling_error(z, y, x):
+    raise FieldSamplingError(f"Field sampled at (depth={z}, lat={y}, lon={x})")
+
+
+def _raise_field_out_of_bound_error(z, y, x):
+    raise FieldOutOfBoundError(f"Field sampled out-of-bound, at (depth={z}, lat={y}, lon={x})")
+
+
+def _raise_field_out_of_bound_surface_error(z, y, x):
+    raise FieldOutOfBoundSurfaceError(f"Field sampled out-of-bound at the surface, at (depth={z}, lat={y}, lon={x})")
 
 
 class TimeExtrapolationError(RuntimeError):
     """Utility error class to propagate erroneous time extrapolation sampling."""
 
-    def __init__(self, time, field=None, msg="allow_time_extrapoltion"):
+    def __init__(self, time, field=None, msg=""):
         if field is not None and field.grid.time_origin and time is not None:
             time = field.grid.time_origin.fulltime(time)
         message = f"{field.name if field else 'Field'} sampled outside time domain at time {time}."
-        if msg == "allow_time_extrapoltion":
-            message += " Try setting allow_time_extrapolation to True"
-        elif msg == "show_time":
-            message += " Try explicitly providing a 'show_time'"
+        if msg == "show_time":
+            message += " Try explicitly providing a 'show_time'."
```

Comment:
We don't have show_time anymore (was part of the old plotting we got rid off years ago, so this is not needed
```suggestion
```